Cargo.toml

```diff
@@ -42,7 +42,7 @@ regex-fancy = ["syntect/regex-fancy"] # Use the rust-only "fancy-regex" engine
 
 [dependencies]
 atty = { version = "0.2.14", optional = true }
-ansi_term = "^0.12.1"
+ansi_term = { version = "0.47", package = "nu-ansi-term" }
```

Comment:
Please specify version as 0.47.0 please (see https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277)

Reply:
done